### DISCUSSION TITLE
How do I add a bloom past to the pipeline?

### DISCUSSION THREAD

**Discussion by rosstheblenderanimator** [bnpr/Malt#538] [Q&A] [answered]
I'm trying to make these head lights glow.
![astonAston MArtinV12 Vantage S 2016 malt ](https://github.com/bnpr/Malt/assets/146353254/6beb0721-0402-49a0-8317-2edbd54d3ec3)

**Comment by pragma37** [comment, marked as answer]
See #315 and # #347.
The attached files use the old deprecated shader nodes, but they still work.
It should be easy to find their current equivalent using the search, though.

**Reply by rosstheblenderanimator** [reply]
I got the car to glow not just the lights. Why is there no equivalent emission node?

**Comment by HSDSZ** [comment]
try post-processing.
